Parsers/ASimUserManagement/Parsers/vimUserManagementMicrosoftSecurityEvent.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
```

Comment:
1. Are Column count same with old and new parser?
2. Are row count coming same with old and new parser?
3. Did we do filter testing for new parser for ASIM and VIM?

Reply:
yes